rpc/src/blobstream.rs

```diff
@@ -0,0 +1,33 @@
+use bytes::Bytes;
+use celestia_types::MerkleProof;
+use jsonrpsee::proc_macros::rpc;
+use prost::bytes;
+
+// DataRootTupleRoot is the root of the merkle tree created
+// from a set of data root tuples.
+pub type DataRootTupleRoot = Bytes;
```

Comment:
if it is a hash, could we alias it to `celestia_types::Hash`?

Reply:
yes can def do that

Reply:
returning hash wont work since its 32 bytes and celestia-node returns a base64 44 byte length string, so `Bytes` does the job here (a user just has to decode it accordingly)